js/main.js

```diff
@@ -100,22 +111,55 @@ function viewSwapping(view) {
   if (view === 'edit-profile') {
     $edit.className = 'edit';
     $profile.className = 'profile hidden';
+
+    if (data.profile.avatarUrl === null || data.profile.avatarUrl === '') {
+      $avatarEditImage.setAttribute('src', 'images/placeholder-image-square.jpg');
+    } else {
+      $avatarEditImage.setAttribute('src', data.profile.avatarUrl);
+    }
+
+    $avatarURL.setAttribute('value', data.profile.avatarUrl);
+    $userName.setAttribute('value', data.profile.username);
+    $fullName.setAttribute('value', data.profile.fullName);
+    $location.setAttribute('value', data.profile.location);
+    $bio.textContent = data.profile.bio;
   }
 
   if (view === 'profile') {
     $profile.className = 'profile';
     $edit.className = 'edit hidden';
+
+    var $container = document.querySelector('.container');
```

Comment:
Do you only have one element with a `container` class, and it is inside the profile container? If so, I would add another class to the element that is unique to that element. The `container` class has special styling correlated to it, so if you ever wanted another container, your code could potentially break.